source/contracts/ITime.sol

```diff
@@ -0,0 +1,16 @@
+pragma solidity 0.4.18;
+
+import 'Controlled.sol';
+import 'libraries/Initializable.sol';
+import 'libraries/ITyped.sol';
+import 'libraries/Ownable.sol';
+import 'libraries/Extractable.sol';
+
+
+contract ITime is Controlled, ITyped, Ownable, Extractable {
```

Comment:
I don't believe this contract needs to be `Controlled` or `ITyped`.  Probably not even `Ownable` either.  `Extractable` is good as a general rule, though users shouldn't _ever_ see the address for this contract so even that seems overkill.

Is there an argument for keeping all of these?

Reply:
Controlled is required for extractable. Also we've generally made any contract that is accesible via lookup controlled so it seems off we'd break that pattern. ITyped is useful to see which provider the current Time contract is. Ownable is at a minimum needed for the testing version of the time provider but we could remove it from the interface

Reply:
Fully convinced of `ITyped`'s value.

I would prefer to see `Ownable` pulled out to the implementation that actually needs it, rather than having it part of the interface since I don't think the interface needs it.

I don't see big value in `Extractable` (and therefore `Controlled`), but I also don't see much downside to including them either... so, meh on those.